merge from master; 5.0.0-beta.2 release (#237)
blackbaud/skyux-layout: CHANGELOG.md

# 5.0.0-beta.2 (2021-08-09)

> - Added bug fixes and features from the `master` branch. [#237](https://github.com/blackbaud/skyux-theme/pull/237)

Reviewer: ```suggestion
- Added bug fixes and features from the `master` branch. [#237](https://github.com/blackbaud/skyux-layout/pull/237)
```

# 4.7.2 (2021-08-06)

> - Fixed the toolbar component to no longer have horizontal padding in modern theme. [#235](https://github.com/blackbaud/skyux-layout/pull/235)

Reviewer: ```suggestion
- Fixed the toolbar component to remove horizontal padding in modern theme. [#235](https://github.com/blackbaud/skyux-layout/pull/235)
```
